Makefile

```diff
@@ -37,7 +38,7 @@ all-srcs:
 .DEFAULT_GOAL := all
 
 .PHONY: all
-all: addlicense impi lint misspell test otelcol
+all: addlicense impi lint misspell test binaries-all-sys
```

Comment:
@tigrannajaryan any reason this was not done initially? Increases build time but by how much? If the desire is to do this at release time how do we ensure it happens?

Reply:
it only releases build time by 60 seconds to build (but not test) the cross-compiled arches.

Reply:
No reason other than that we started with Linux. I believe the increase in CI build time is acceptable but this also applies to default local builds with `make`, which maintainers do a lot.

I'd prefer `binaries-all-sys` to be only in the `ci` path and locally (for the default make target) we only build one architecture. It will save a lot of time when in local development cycle when you want to make sure it compiles and passes tests when iterating. @lizthegrey I think it should be as easy as just listing `binaries-all-sys` as a dependency in `ci` (IIRC `make` won't even run dependencies twice since every dependency is evaluated once - which is exactly what we need).

Reply:
Sure thing, can do.